core/src/sigverify.rs

```diff
@@ -128,7 +129,7 @@ impl SigVerifier for TransactionSigVerifier {
     ) -> Result<(), SigVerifyServiceError<Self::SendType>> {
         let tracer_packet_stats_to_send = std::mem::take(&mut self.tracer_packet_stats);
         self.packet_sender.send(BankingPacketBatch::new((
-            packet_batches,
+            packet_batches.into_iter().map(Arc::new).collect(),
```

Comment:
not happy with the conversion being here, it'd probably be better if we could move it up to arc-wrapping when SV receives these into the vec initially.

but i couldn't figure out a good way to do that because `PacketBatch` and `dedup` stuff are re-used for other stuff in the code, like shreds. we don't need to wrap those in an Arc, so don't want to add the overhead there. and traits for `Arc` are a bit lacking since there's seemingly not a trait I could use to get mutable reference to arc's interior.

Would be nice if there was some sort of `MutThenToArc` that safely allows me to mutate the arc contents, and then just cast my `Vec<MutThenToArc<T>>` to a `Vec<Arc<T>>` at the end of my mutation without doing any re-allocations.